docs/src/pages/components/date-range-picker/CustomDateRangePickerDay.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import TextField from '@material-ui/core/TextField';
+import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
+import LocalizaitonProvider from '@material-ui/lab/LocalizationProvider';
+import DateRangePicker, { DateRange } from '@material-ui/lab/DateRangePicker';
+import DateRangePickerDay, {
+  DateRangePickerDayProps,
+} from '@material-ui/lab/DateRangePickerDay';
+import clsx from 'clsx';
+
+const useStyles = makeStyles((theme) => ({
+  highlight: {
+    borderRadius: 0,
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.common.white,
+    '&:hover, &:focus': {
+      backgroundColor: theme.palette.primary.dark,
+    },
+  },
+  firstHighlight: {
+    borderTopLeftRadius: '50%',
+    borderBottomLeftRadius: '50%',
+  },
+  endHighlight: {
+    borderTopRightRadius: '50%',
+    borderBottomRightRadius: '50%',
+  },
+}));
+
+export default function CustomDateRangePickerDay() {
+  const classes = useStyles();
+  const [selectedDate, handleDateChange] = React.useState<DateRange<Date>>([
```

Comment:
What do you think about following the naming convention of the other demos?

```suggestion
  const [value, setValue] = React.useState<DateRange<Date>>([
```

Reply:
I followed a convention.

Reply:
With a grep of `React.useState<Date | null>`, I can count:

- `value` x19
- `selectedDate` x3
- `date` x1

It seems that, so far (if we ignore what Dmitiry was doing), we have used the name of the prop as the default (unless there are multiple states) name for the state.

Reply:
Let's do this in a follow-up. Marking this review for future reference when I'm being told we "move fast".